src/sage/rings/polynomial/multi_polynomial_sequence.py

```diff
@@ -771,16 +759,16 @@ def coefficient_matrix(self, sparse=True):
         #construct dictionary for fast lookups
         v = dict( zip( m , range(len(m)) ) )
 
-        from sage.matrix.constructor import Matrix
-
         A = Matrix( R.base_ring(), nf, nm, sparse=sparse )
 
         for x in range( nf ):
             poly = f[x]
             for y in poly.monomials():
                 A[ x , v[y] ] = poly.monomial_coefficient(y)
 
-        return A, Matrix(R,nm,1,m)
+        deprecation(37027,"The PolynomialSequence.coeffient_matrix() now returns a vector of monomials instead of a 3 by 1 matrix")
```

Comment:
You certainly do not want to hardcode the size of the matrix.

Reply:
Ohh ! Ya, I think I have not pushed latest commits, let me do it. Actually, this is from my previous commit and I forgot to push latest one.